omod/src/main/webapp/resources/js/custom/Custom.js

```diff
@@ -178,195 +215,240 @@ $(function(){
                 });
             },
             error: function(msg){
-                    alert("Internal server error : while getting location list ");
-                }
+                alert("Internal server error : while getting location list ");
             }
-        );
-        $.ajax({
+        }
+    );
+    $.ajax({
```

Comment:
I think it's much easier to use jQuery.get here :)

Look up the documentation for this.